src/platform/packages/shared/kbn-unified-tabs/src/components/tabs_bar/optional_droppable.tsx

```diff
@@ -46,19 +48,44 @@ interface DroppableWrapperProps {
 export const OptionalDroppable: FC<DroppableWrapperProps> = ({
   children,
   disableDragAndDrop,
-  onDragEnd,
+  onDragStart: originalOnDragStart,
+  onDragEnd: originalOnDragEnd,
 }) => {
   const { euiTheme } = useEuiTheme();
+  const [isDragging, setIsDragging] = useState(false);
 
-  const draggingCss = css`
-    .unifiedTabs__tabWithBackground:not(.unifiedTabs__tabWithBackground--selected) {
-      background-color: transparent;
+  const draggingCss = useMemo(
+    () => css`
+      .unifiedTabs__tabWithBackground {
+        pointer-events: none;
```

Comment:
I also added `pointer-events: none;` to other tabs when dragging so things like hover styles and inline buttons aren't displayed.